packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -123,7 +125,9 @@ class PortLink(BaseModel):
 class DownloadLink(BaseModel):
     """I/O port type to hold a generic download link to a file (e.g. S3 pre-signed link, etc)"""
 
-    download_link: Annotated[str, AnyUrl] = Field(..., alias="downloadLink")
+    download_link: Annotated[
+        str, BeforeValidator(lambda x: str(_ANY_URL_ADAPTER.validate_python(x)))
```

Comment:
I am surprised about thisone. There is not a type that pydantic knows how to parse in the line of `download_link: URL`?

Reply:
This change generated a lot of pain: https://docs.pydantic.dev/latest/migration/#url-and-dsn-types-in-pydanticnetworks-no-longer-inherit-from-str